src/macros.rs

```diff
@@ -15,6 +15,12 @@ macro_rules! assert_buffer {
     }};
 }
 
+macro_rules! assert_buffer_eq {
```

Comment:
What is this for?

Reply:
Not sure honestly. I kept it from the original secp256k1 PR. Better remove it, I think

Reply:
Looks completely unused yeah

Reply:
Removed

Reply:
Not yet?